src/Polysemy/ConstraintAbsorber.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE AllowAmbiguousTypes         #-}
+{-# LANGUAGE ConstraintKinds             #-}
+{-# LANGUAGE FlexibleInstances           #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving  #-}
+{-# LANGUAGE MultiParamTypeClasses       #-}
+{-# LANGUAGE ScopedTypeVariables         #-}
```

Comment:
STV and TypeFamilies are already enabled on a project-wide basis. Please remove from here and other files.